src/briefcase/bootstraps/toga.py

```diff
@@ -65,17 +65,33 @@ def pyproject_table_linux(self):
 
     def pyproject_table_linux_system_debian(self):
         return """\
+requires = [
+    # PyGObject 3.52.1 enforces a requirement on libgirepository-2.0-dev. This library
+    # isn't available on Debian 13/Ubuntu 22.04. If you need to support those (or older)
+    # releases, uncomment this version pin. See beeware/toga#3143.
+    # "pygobject < 3.52.1",
+]
+
 system_requires = [
     # Needed to compile pycairo wheel
     "libcairo2-dev",
-    # Needed to compile PyGObject wheel
-    "libgirepository1.0-dev",
+    # One of the following two packages are needed to compile PyGObject wheel.
+    # Debian 13/Ubuntu 22.04 only has the older 1.0 version of the library.
+    # If you use the older version, you'll also need to apply the pygobject pin
+    # (see the linux requires table). See beeware/toga#3143.
+    "libgirepository-2.0-dev",
+    # "libgirepository1.0-dev",
 ]
 
 system_runtime_requires = [
     # Needed to provide GTK and its GI bindings
     "gir1.2-gtk-3.0",
-    "libgirepository-1.0-1",
+    # One of the following two packages are needed to use PyGObject at runtime.
+    # Debian 13/Ubuntu 22.04 only has the older 1.0 version of the library.
+    # If you use the older version, you'll also need to apply the pygobject pin
+    # (see the linux requires table). See beeware/toga#3143.
+    "libgirepository-2.0-0",
+    # "libgirepository-1.0-1",
     # Dependencies that GTK looks for at runtime
     "libcanberra-gtk3-module",
     # Needed to provide WebKit2 at runtime
```

Comment:
We can remove the note below about Debian 11 and 20.04; they aren't supported anymore anyway.

Reply:
Oh, I guess it's just Ubuntu 20.04 that could be removed; looks like we still support Debian 11. Ubuntu 22.04 doesn't work anymore, though, since it uses Python 3.8.

We can worry about this a different time, though; more pressing matters right now.